CHANGELOG.md

```diff
@@ -3,6 +3,19 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v2.0.0
```

Comment:
New version should be v1.10.0, as this is only a minor version change.

Also needs this to be updated in the `package.json`